source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -127,6 +127,32 @@ envoy::extensions::filters::network::http_connection_manager::v3::HttpConnection
       KEEP_UNCHANGED;
 }
 
+Http::HeaderValidatorFactoryPtr
+createHeaderValidatorFactory([[maybe_unused]] const envoy::extensions::filters::network::
+                                 http_connection_manager::v3::HttpConnectionManager& config,
+                             [[maybe_unused]] Server::Configuration::FactoryContext& context) {
+
+  Http::HeaderValidatorFactoryPtr header_validator_factory;
+#ifdef ENVOY_ENABLE_UHV
```

Comment:
this is only ifdef'd out until we have runtime controls of underlying header validation correct?

Should we throw an exception if it's configured on but not supported due to compile time options?

Reply:
Done